blocks/bricks/interfaces.py

```diff
@@ -141,31 +141,61 @@ class Initializable(RNGMixin, Brick):
 
     """
     has_biases = True
+    initialization_roles = set([])
 
     @lazy()
-    def __init__(self, weights_init=None, biases_init=None, use_bias=True,
-                 seed=None, **kwargs):
+    def __init__(self,
```

Comment:
yeah, will go thorough formatting issues at the end. This was just to open the discussion about the way to solve this issues.